### ISSUE TITLE
Storyboard doesn't contain a view controller with identifier 'SignIn'

### ISSUE THREAD

**Issue by eimantas** [aws-amplify/aws-sdk-ios#1064]
**Describe the bug**
Application crashes when presenting a sign in view controller.

**To Reproduce**
1. Create new single view application
2. Add AWS SDK for iOS and import these modules into `AppDelegate`:
  - `AWSCore`
  - `AWSAuthCore`
  - `AWSAuthUI`
3. The `AppDelegate.swift` should look like this:

```swift
import UIKit

import AWSCore
import AWSAuthCore
import AWSAuthUI

@UIApplicationMain
class AppDelegate: UIResponder, UIApplicationDelegate {

    var window: UIWindow?

    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {

        let navC = UINavigationController(nibName: nil, bundle: nil)
        window?.rootViewController = navC

        AWSAuthUIViewController.presentViewController(with: navC, configuration: nil) { (provider, maybeError) in
            //
        }

        return true
    }
}
```

**Expected behavior**
Application is started with AWS sign in screen.

**Environment(please complete the following information):**
 - SDK Version: 2.6.32
 - Dependency Manager: Carthage
 - Swift Version : 4.2

**Device Information (please complete the following information):**
 - Device: iPhone X🅁 Simulator
 - iOS Version: 12.0

**Additional context**

Stacktrace of the crash:
```
2018-10-18 07:28:27.294850+0300 Project[58281:13784081] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Storyboard (<UIStoryboard: 0x600000656cc0>) doesn't contain a view controller with identifier 'SignIn''
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000103f5b29b __exceptionPreprocess + 331
	1   libobjc.A.dylib                     0x0000000102a5b735 objc_exception_throw + 48
	2   UIKitCore                           0x0000000109fd8d0c -[UIStoryboard instantiateInitialViewController] + 0
	3   AWSAuthUI                           0x0000000102411094 +[AWSSignInViewController getViewControllerWithName:storyboard:] + 80
	4   AWSAuthUI                           0x00000001024110f3 +[AWSSignInViewController getAWSSignInViewControllerWithconfiguration:] + 49
	5   AWSAuthUI                           0x0000000102411d5c __105+[AWSAuthUIViewController presentViewControllerWithNavigationController:configuration:completionHandler:]_block_invoke + 110
	6   libdispatch.dylib                   0x000000010509d5d1 _dispatch_call_block_and_release + 12
	7   libdispatch.dylib                   0x000000010509e63e _dispatch_client_callout + 8
	8   libdispatch.dylib                   0x00000001050ab9d6 _dispatch_main_queue_callback_4CF + 1541
	9   CoreFoundation                      0x0000000103ebe7f9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	10  CoreFoundation                      0x0000000103eb8e86 __CFRunLoopRun + 2342
	11  CoreFoundation                      0x0000000103eb8221 CFRunLoopRunSpecific + 625
	12  GraphicsServices                    0x00000001096001dd GSEventRunModal + 62
	13  UIKitCore                           0x0000000109ae2115 UIApplicationMain + 140
	14  AWSome                              0x0000000101cfef87 main + 71
	15  libdyld.dylib                       0x0000000105114551 start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by rohandubal** [comment]
Hello @eimantas 

I am working on a fix to resolve this issue and it will be fixed in a future SDK release.

Thanks,
Rohan

**Comment by rohandubal** [comment]
Hello,

This issue is resolved in `2.7.4` SDK release. Please let us know if you continue to face issues.

Thanks,
Rohan

**Comment by eimantas** [comment]
Thanks a heap, Rohan! I will test this. Closing the issue for now.
